src/components/atomic/Checkbox.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { InputHTMLAttributes } from "react";
+import classNames from "classnames";
+import { Text } from "../../components/atomic";
+
+type CheckboxProps = InputHTMLAttributes<HTMLInputElement> & {
+  labeltext?: string;
+};
+
+const Checkbox = ({ className = "", ...props }: CheckboxProps) => {
+  const checkboxStyles = classNames({
+    ["h-5 w-5 border-2 border-primary rounded appearance-none \
+      hover:border-3 active:border-darkblue active:border-3\
```

Comment:
Instead of border-3 (thickening on hover), maybe we could try darkening? Border-thickening is slightly jarring.

- [ ] (Or however Github does their checkboxes haha)